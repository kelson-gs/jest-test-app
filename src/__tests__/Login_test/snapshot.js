import { render } from "@testing-library/react"
import Login from '../../pages/login' 

it("renders loginpage unchanged", () => {
  const { container } = render(<Login />)
  expect(container).toMatchSnapshot()
})
