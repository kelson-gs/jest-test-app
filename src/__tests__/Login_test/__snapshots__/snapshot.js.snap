// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders loginpage unchanged 1`] = `
<div>
  <div>
    <h1>
      Login...
    </h1>
    <form>
      <input
        type="email"
      />
      <input
        type="password"
      />
      <button>
        Entrar
      </button>
    </form>
  </div>
</div>
`;
